cmd/gateway/azure/gateway-azure.go

```diff
@@ -125,6 +131,19 @@ func azureGatewayMain(ctx *cli.Context) {
 	minio.StartGateway(ctx, &Azure{host})
 }
 
+func getUploadChunkSizeFromEnv(envvar string, defaultValue int) int {
+	envChunkSize := os.Getenv(envvar)
+	i, err := strconv.ParseFloat(envChunkSize, 64)
+	if err != nil {
+		return defaultValue
```

Comment:
Perhaps use `logger.LogIf(ctx, err)` to indicate that we are proceeding to use default value?

Reply:
Done